controllers/etcd_controller_test.go

```diff
@@ -602,6 +605,214 @@ var _ = Describe("Druid", func() {
+
+			Eventually(func() error {
+				return c.Get(context.TODO(), types.NamespacedName{
+					Name:      instance.Name,
+					Namespace: instance.Namespace,
+				}, instance)
+			}, timeout, pollingInterval).Should(BeNil())
+
+			sts = &appsv1.StatefulSet{}
+			// No StatefulSet has been created by controller as even number of replicas are not allowed
+			Eventually(func() error {
+				return c.Get(context.TODO(), types.NamespacedName{
+					Name:      instance.Name,
+					Namespace: instance.Namespace,
+				}, sts)
+			}, timeout, pollingInterval).Should(matchers.BeNotFoundError())
+
+			// Update replicas in ETCD resource with even number
+			Expect(kutil.TryUpdate(context.TODO(), retry.DefaultBackoff, c, instance, func() error {
+				instance.Spec.Replicas = 4
+				return nil
+			})).To(Succeed())
+
+			Eventually(func() error {
+				return c.Get(context.TODO(), types.NamespacedName{
+					Name:      instance.Name,
+					Namespace: instance.Namespace,
+				}, instance)
+			}, timeout, pollingInterval).Should(BeNil())
+
+			// No StatefulSet has been created by controller as even number of replicas are not allowed
+			sts = &appsv1.StatefulSet{}
+			Eventually(func() error {
+				return c.Get(context.TODO(), types.NamespacedName{
+					Name:      instance.Name,
+					Namespace: instance.Namespace,
+				}, sts)
+			}, timeout, pollingInterval).Should(matchers.BeNotFoundError())
+
+			// No Service has been created by controller as even number of replicas are not allowed
+			svc = &corev1.Service{}
+			Expect(c.Get(context.TODO(), types.NamespacedName{
+				Name:      fmt.Sprintf("%s-client", instance.Name),
+				Namespace: instance.Namespace,
+			}, svc)).Should(matchers.BeNotFoundError())
+			svc = nil
+		})
+		It("configmaps are mounted properly when ETCD replicas are odd number", func() {
+			// Update replicas in ETCD resource with 1
+			instance.Spec.Replicas = 1
+			Expect(c.Create(context.TODO(), instance)).To(Succeed())
+
+			sts = &appsv1.StatefulSet{}
+			// StatefulSet has been created by controller
+			Eventually(func() error {
+				if err := c.Get(context.TODO(), types.NamespacedName{
+					Name:      instance.Name,
+					Namespace: instance.Namespace,
+				}, sts); err != nil {
+					return err
+				}
+
+				if *sts.Spec.Replicas != int32(instance.Spec.Replicas) {
+					return fmt.Errorf("Statefulset replicas not updated to: %d", instance.Spec.Replicas)
+				}
+
+				// configmap volume in statefulset container should be mounted in /var/etcd/config/
+				if sts.Spec.Template.Spec.Containers[0].VolumeMounts[1].MountPath != "/var/etcd/config/" {
+					return fmt.Errorf("Etcd config for ETCD container is not properly updated")
+				}
+
+				if sts.Spec.Template.Spec.Containers[1].VolumeMounts[1].MountPath != "/var/etcd/config/" {
+					return fmt.Errorf("Etcd config ETCD backup container is not properly updated")
+				}
+
+				return nil
+			}, timeout, pollingInterval).Should(BeNil())
```

Comment:
```suggestion
			}, timeout, pollingInterval).Should(Succeed())
```
Reads better this way.